### PR TITLE
docs: update readme from last commit

### PR DESCRIPTION
**Pull request by rossiam**
Forgot to run `npm run readme` for last commit.

## Checklist

- [x] I have read the **[CONTRIBUTING](../CONTRIBUTING.md)** document
- [ ] Any required documentation has been added
- [ ] My code follows the code style of this project (`npm run lint` produces no warnings/errors)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -2493,7 +2493,7 @@ FLAGS
                         or an ISO time string
   -B, --before=<value>  return events older than than this timestamp, expressed as an epoch time in milliseconds or an
                         ISO time string
-  -L, --limit=<value>   maximum number of events to return, defaults to 20
+  -L, --limit=<value>   [default: 20] maximum number of events to return, defaults to 20
```
Comment (Sitlintac) [inline packages/cli/README.md:2496]: `defaults to 20` is redundant now.

File: packages/cli/README.md [hunk 2]
```diff
@@ -3756,7 +3756,7 @@ FLAGS
...
```
Comment (Sitlintac) [inline packages/cli/README.md:3759]: Same as above

**Review by Sitlintac — APPROVED** [review]

File: packages/cli/README.md (hunk 1 above)
Comment (rossiam) [reply packages/cli/README.md]: Good catch. Fixed.

File: packages/cli/README.md (hunk 2 above)
Comment (rossiam) [reply packages/cli/README.md]: Fixed

**Pull request merged**